doc/releases/changelog-dev.md

```diff
@@ -89,6 +89,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Finite shot circuits with a `qml.probs` measurement, both with a `wires` or `op` argument, can now be compiled with `jax.jit`.
```

Comment:
```suggestion
* Finite shot circuits with a `qml.probs` measurement, both with a `wires` or `op` argument, can now be fully compiled with `jax.jit` without use of inadvertent pure callbacks.
```


The current phrasing kinda makes it sound like probs+jax.jit straight up didn't work.  But it did; it was just less than inefficient.

Reply:
It straight up didn't work because of `qml.math.unique`. It now works and is free of callbacks.